src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -142,13 +147,16 @@ object SbtBundle extends AutoPlugin {
   override def trigger = AllRequirements
 
   override def projectSettings = Seq(
-    system := (packageName in Universal).value,
+    compatibilityVersion := version.value.takeWhile(_ != '.'),
```

Comment:
@huntc - is there a reason why we lose the `.` ?

We can't tell the difference between `1.0.5` vs `10.5` - it becomes `105`